distribution/docs/src/main/resources/content/_running/console-commands.adoc

```diff
@@ -1,611 +1,20 @@
 :title: Console Commands
-:type: maintaining
+:type: maintainingIntro
 :status: published
 :summary: Introduction to console commands.
 :order: 00
 
-==== Console Commands
+== {title}
 
-Once the distribution has started, users will have access to a powerful command line console, the ${command-console}.
+Once the distribution has started, administrators will have access to a powerful command line console, the ${command-console}.
 This ${command-console} can be used to manage services, install new features and applications, and manage the state of the system.
 
-===== Accessing the System Console
-
-The Command Line Console is the console that is available to the user when the distribution is started manually.
-It may also be accessed by using the `bin/client.bat` or `bin/client.sh` scripts.
+The Command Line Console is available to the user when the distribution is started manually or may also be accessed by using the `bin/client.bat` or `bin/client.sh` scripts.
```

Comment:
A note could be added here about how to see a list of all available console commands. I think it might be tab in the karaf console, but I can't remember.

Reply:
Pretty sure that's it too, but will confirm